src/moi_oracle.jl

```diff
@@ -292,6 +292,45 @@ function dicg_maximum_step(lmo::MathOptLMO{OT}, direction, x) where {OT}
     end
 end
 
+function is_inface_feasible(lmo::MathOptLMO{OT}, a, x;) where {OT}
+	variables = MOI.get(lmo.o, MOI.ListOfVariableIndices())
+	valvar(f) = x[f.value]
+	valvar_(f) = a[f.value]
```

Comment:
Maybe call this `valvar_away` or something. Having an underscore last is not good style and easy to miss.